### PR TITLE
New: add rule no-plusplus

### PR DESCRIPTION
**Pull request by aladdin-add**

### CODE REVIEW

File: lib/rules/no-plusplus.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const rule = utils.getFixableRule("no-plusplus");
+const fixableTypes = new Set(["ExpressionStatement", "ForStatement"]);
+
+module.exports = ruleComposer.mapReports(
+    rule,
+    problem => {
+        const node = problem.node;
+        const op = node.operator === "++" ? "+=1" : "-=1";
+
+        problem.fix = function(fixer) {
```
Comment (g-plane) [inline lib/rules/no-plusplus.js]: Non-blocking: we can use arrow function.

Comment (aladdin-add) [reply lib/rules/no-plusplus.js]: [image](https://user-images.githubusercontent.com/13050025/52550628-cff97800-2e13-11e9-82b3-b35f07eb29d6.png)

we got an error here, but it seems a bug to me.(I've add a paren on the expression.)

Comment (g-plane) [reply lib/rules/no-plusplus.js]: It's defined at `eslint-config-eslint`, not a parsing error. Use `function` back or disable that rule with a comment?

Comment (aladdin-add) [reply lib/rules/no-plusplus.js]: IMO, the rule should not report in this case, you think?

Comment (g-plane) [reply lib/rules/no-plusplus.js]: Add `allowParens` option for this rule in global ESLint config.

Comment (aladdin-add) [reply lib/rules/no-plusplus.js]: the option might need to be enabled by default, can you file an issue in eslint repo?

Comment (g-plane) [reply lib/rules/no-plusplus.js]: Hmm...I think they may have their opinion, so I'm not going to do that.

**Pull request merged**